php/containers.json

```diff
@@ -217,7 +217,11 @@
           "writeable": true
         }
       ],
-      "restart": "unless-stopped"
+      "restart": "unless-stopped",
+      "nextcloud_exec_commands": [
+        "echo 'Activating collabora config...'",
+        "php /var/www/html/occ richdocuments:activate-config"
```

Comment:
this will not work in manual-install and co. will need to thing about a solution. Maybe actually as env in containers.json?